src/org/opendatakit/briefcase/ui/export/components/ConfigurationDialogForm.java

```diff
@@ -0,0 +1,189 @@
+package org.opendatakit.briefcase.ui.export.components;
+
+import static java.awt.event.KeyEvent.VK_ESCAPE;
+import static javax.swing.JComponent.WHEN_ANCESTOR_OF_FOCUSED_COMPONENT;
+import static javax.swing.KeyStroke.getKeyStroke;
+
+import java.awt.Component;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JDialog;
+import javax.swing.JPanel;
+import org.opendatakit.briefcase.ui.reused.WindowAdapterBuilder;
+
+@SuppressWarnings("checkstyle:MethodName")
+public class ConfigurationDialogForm extends JDialog {
+  private JPanel dialog;
+  private JButton okButton;
+  private JButton cancelButton;
+  private JButton removeButton;
+  private JPanel rightActions;
+  private JPanel leftActions;
+  private JPanel actions;
+  private ConfigurationPanelForm configurationPanelForm;
+
+  ConfigurationDialogForm(ConfigurationPanelForm form) {
+    configurationPanelForm = form;
+    $$$setupUI$$$();
+    setContentPane(dialog);
+    setModal(true);
+    getRootPane().setDefaultButton(okButton);
+
+    okButton.addActionListener(e -> dispose());
+    removeButton.addActionListener(e -> dispose());
+    cancelButton.addActionListener(e -> dispose());
+
+    setDefaultCloseOperation(DO_NOTHING_ON_CLOSE);
+
+    addWindowListener(new WindowAdapterBuilder().onClosing(e -> dispose()).build());
+
+    dialog.registerKeyboardAction(e -> dispose(), getKeyStroke(VK_ESCAPE, 0), WHEN_ANCESTOR_OF_FOCUSED_COMPONENT);
```

Comment:
I’m not too concerned about it, but in case you’re not aware, registerKeyboardAction has this comment:

**This method is now obsolete, please use a combination of getActionMap() and getInputMap() for similar behavior.**

Reply:
Honestly, I'd wait for a `@Deprecated` annotation on that method to change it... 

Also, this line was generated by IntelliJ's GUI designer... I'm guessing that it's safe to use.

Let's keep it :)